src/main/java/uk/gov/companieshouse/model/dto/dissolution/DissolutionDirectorPatchResponse.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.model.dto.dissolution;
+
+public class DissolutionDirectorPatchResponse {
```

Comment:
can any inheritance be used here between DissolutionCreateResponse or others? If so, consider refactoring the mappers to reduce redefining the generateLinks method.